src/components/pages/bc-new-wallet-link.ts

```diff
@@ -0,0 +1,105 @@
+import {customElement} from 'lit/decorators.js';
+import {BitcoinConnectElement} from '../BitcoinConnectElement';
+import {withTwind} from '../twind/withTwind';
+import {html} from 'lit';
+import {classes} from '../css/classes';
+import '../internal/bci-button';
+import {albyIcon} from '../icons/connectors/albyIcon';
+
+@customElement('bc-new-wallet-link')
```

Comment:
can you rename this component and file to `bc-new-wallet` ? I don't think we need the "-link" suffix.